Autopilot/PathManager/Inc/landingTakeoffManager.hpp

```diff
@@ -0,0 +1,167 @@
+#pragma once
+
+/* Related Confluence Pages
+    https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/1607630921/Landing+and+Takeoff
+*/
+
+#include "waypointManager.hpp"
+#include "vectorClass.hpp"
+#include "AutoSteer.hpp"
+#include "SensorFusion.hpp"
+#include "pathDatatypes.hpp"
+
+enum _FlightStage{ROLL=0, CLIMB, CRUISING, TRANSITION, SLOPE, FLARE, DECRAB, TOUCHDOWN};
+
+/************************
+    LANDING SECTION
+************************/
+
+constexpr static double DISTANCE_OF_LANDING {10}; //in meters
+constexpr static double METERS_PER_DEG_LAT {111320.0};
+constexpr static double ANGLE_OF_LANDING {5}; //in degrees
+constexpr static int STALL_SPEED_NO_PACKAGE {2}; //in m/s
+constexpr static int STALL_SPEED_WITH_PACKAGE {4}; //in m/s
+constexpr static double PI {3.14159265358979323846};
+constexpr static int FLARE_ALTITUDE {2}; //in meters
+constexpr static double DECRAB_ALTITUDE {0.7}; //in meters
+constexpr static double TOUCHDOWN_ALTITUDE {0.05}; //in meters
+constexpr static double LATITUDE_LONGITUDE_CONVERSION_CONSTANT {40075000.0};
+
+struct _LandingPath //this structure is used to hold the waypoint dat of the landing path, when it is created in the transition stage
+{
+    _PathData stoppingPoint;
+    _PathData aimingPoint;
+    _PathData intersectionPoint;
```

Comment:
hate to be nitpicky, but what I really meant was explaining things like what the intersection point is in a few words would make things much more understandable.